### DISCUSSION TITLE
A problem with the dashboard displaying

### DISCUSSION THREAD

**Discussion by lamld203844** [Lightning-AI/dl-fundamentals#93] [Lecture videos and quizzes] [answered]
It seems there is a problem with the dashboard. The items in the list are not been in order recently.
![lightningAI-discuss](https://github.com/Lightning-AI/dl-fundamentals/assets/78334320/3c4389fc-e3a9-4d32-97cd-77cc302e82dd)

**Comment by rasbt** [comment, marked as answer]
Thanks for the note @lamld203844 ! Just pinged our web dev and hope to get that fixed soon!

**Reply by rasbt** [reply]
Should be fixed now. Thanks again for the note.
